# information about the threshold value
# custom threshold_status_t
	bool x_axis  # (0 = not met | 1 = met) indicates if threshold on the X axis was met
	bool y_axis  # (0 = not met | 1 = met) indicates if threshold on the Y axis was met
	bool z_axis  # (0 = not met | 1 = met) indicates if threshold on the Z axis was met
	bool enabled  # (0 = Disabled | 1 = Enabled) indicates if the threshold check is enabled
